review.py

```diff
@@ -48,19 +53,59 @@ def href_to_shortname(href):
 
 def maybe_create_prs(shortnames):
     for shortname in shortnames:
-        os.chdir("../{}".format(shortname))
+        os.chdir(f"../{shortname}")
         maybe_create_pr(shortname)
         os.chdir(".")
 
+def replace_rd_pointer(shortname, contents, path_month):
+    if shortname != "html":
+        return re.sub(
+            "Text Macro: LATESTRD [0-9]+-[0-9]+",
+            f"Text Macro: LATESTRD {path_month}",
+            contents
+        )
+
+    return re.sub(
+        "<a href=\"/review-drafts/[0-9]+-[0-9]+/\">",
+        f"<a href=\"/review-drafts/{path_month}/\">",
+        contents
+    )
+
+def add_date_to_rd(shortname, contents, today):
+    if shortname != "html":
+        metadata_date = today.strftime("%Y-%m-%d")
+        return re.sub(
+            "Group: WHATWG",
+            f"Group: WHATWG\nDate: {metadata_date}",
+            contents
+        )
+
+    title_date = today.strftime("%B %Y")
+    pubdate = today.strftime("%d %B %Y")
+    with_title_date = re.sub(
+        "<title w-nodev>HTML Standard</title>",
+        f"<title w-nodev>HTML Standard Review Draft {title_date}</title>",
+        contents
+    )
+
+    # This intentionally removes the <span class="pubdate"> since otherwise Wattsi would put in the build date.
```

Comment:
This is generally the case because of https://github.com/whatwg/whatwg.org/issues/348 right? (Although maybe for Bikeshed files it's worse.)

Reply:
This line only applies to HTML (Wattsi), since only Wattsi has special handling for `<span class="pubdate">`.

I also don't really understand the linked issue... I'll comment over there.